inventory-service/src/test/java/com/petclinic/inventoryservice/presentationlayer/InventoryControllerIntegrationTest.java

```diff
@@ -453,6 +472,131 @@ void deleteAllInventories_ShouldDeleteAllInventoriesAndAssociatedProducts() {
+                .productDescription("Updated Product Description")
+                .productPrice(150.00)
+                .productQuantity(20)
+                .build();
+
+        // Act and assert
+        webTestClient
+                .put()
+                .uri("/inventory/" + validInventoryId + "/products/" + validProductId)
+                .accept(MediaType.APPLICATION_JSON)
+                .contentType(MediaType.APPLICATION_JSON)
+                .bodyValue(productRequestDTO)
+                .exchange()
+                .expectStatus().isOk()
+                .expectHeader().contentType(MediaType.APPLICATION_JSON)
+                .expectBody(ProductResponseDTO.class)
+                .value(updatedProduct -> {
+                    assertNotNull(updatedProduct);
+                    assertEquals(validProductId, updatedProduct.getProductId());
+                    assertEquals(productRequestDTO.getProductName(), updatedProduct.getProductName());
+                    assertEquals(productRequestDTO.getProductDescription(), updatedProduct.getProductDescription());
+                    assertEquals(productRequestDTO.getProductPrice(), updatedProduct.getProductPrice());
+                    assertEquals(productRequestDTO.getProductQuantity(), updatedProduct.getProductQuantity());
+                });
+    }
+
+     */
+
+
+
+
+    /*
+    @Test
+    public void testUpdateProductInInventory_ShouldSucceed() {
+        String inventoryId = "1";
+        String productId = UUID.randomUUID().toString();
+        ProductRequestDTO productRequestDTO = ProductRequestDTO.builder()
+                .productName("Updated Benzodiazepines")
+                .productDescription("Updated Sedative Medication")
+                .productPrice(150.00)
+                .productQuantity(20)
+                .build();
+        webTestClient.put()
+                .uri("/inventories/{inventoryId}/products/{productId}", inventoryId, productId)
+                .accept(MediaType.APPLICATION_JSON)
+                .contentType(MediaType.APPLICATION_JSON)
+                .bodyValue(productRequestDTO)
+                .exchange()
+                .expectStatus().isOk()
+                .expectHeader().contentType(MediaType.APPLICATION_JSON)
+                .expectBody(ProductResponseDTO.class)
+                .value(productResponseDTO -> {
+                    assertNotNull(productResponseDTO);
+                    assertEquals(productId, productResponseDTO.getProductId());
+                    assertEquals(productRequestDTO.getProductName(), productResponseDTO.getProductName());
+                    assertEquals(productRequestDTO.getProductDescription(), productResponseDTO.getProductDescription());
+                    assertEquals(productRequestDTO.getProductPrice(), productResponseDTO.getProductPrice());
+                    assertEquals(productRequestDTO.getProductQuantity(), productResponseDTO.getProductQuantity());
+                });
+    }
```

Comment:
remove ALL this commented code their should be no commented code in production code